Engine/DataFeeds/LiveTradingDataFeed.cs

```diff
@@ -474,7 +474,7 @@ private Subscription CreateUniverseSubscription(SubscriptionRequest request)
 
                 var factory = new LiveCustomDataSubscriptionEnumeratorFactory(_timeProvider);
                 var enumeratorStack = factory.CreateEnumerator(request, _dataProvider);
-                enumerator = new BaseDataCollectionAggregatorEnumerator(enumeratorStack, config.Symbol);
+                enumerator = new BaseDataCollectionAggregatorEnumerator(enumeratorStack, config.Symbol, liveMode:true);
```

Comment:
I wholeheartedly approve of the `liveMode:true` syntax -- it's nice to give names to `bool` parameters 👍